.github/pull_request_template.md

```diff
@@ -7,7 +7,7 @@ Closes # (if applicable).
 
 - [ ] I tested my contribution locally and it works as intended.
 - [ ] Code and workflow changes are sufficiently documented.
-- [ ] Changed dependencies are added to `envs/environment.yaml`.
+- [ ] Changed dependencies are added to `pixi.toml`.
```

Comment:
Should the file be manually edited or similar to `uv` using `uv add` (what's the `pixi` equivalent?)